release/nightly_tests/dataset/batch_inference_benchmark.py

```diff
@@ -0,0 +1,120 @@
+import argparse
+import io
+import uuid
+from typing import Any, Dict
+
+import boto3
+import numpy as np
+import pandas as pd
+import torch
+from benchmark import Benchmark
+from PIL import Image
+from torchvision.models import ResNet50_Weights, resnet50
+
+import ray
+from ray.data import ActorPoolStrategy
+
+BUCKET = "anyscale-imagenet"
+# This Parquet file contains the keys of images in the 'anyscale-imagenet' bucket.
+METADATA_PATH = "s3://anyscale-imagenet/metadata.parquet"
+
+# Largest batch that can fit on a T4.
+BATCH_SIZE = 900
+
+WRITE_PATH = f"s3://ray-data-write-benchmark/{uuid.uuid4().hex}"
```

Comment:
unrelated to PR - how long do we keep this around?

Reply:
I think it's like 3 days. We have a retention policy setup